packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -153,6 +155,22 @@ class Focus extends StatefulWidget {
   // when then widget is updated.
   bool get _usingExternalFocus => false;
 
+  /// Returns the parent node to use when reparenting the [focusNode] for this
```

Comment:
nit: maybe omit "Returns"? Most people will be reading this doc to figure out if they should set the parentNode property...

Reply:
Maybe say: "Optional parent node to use when..." (to really stress that most people don't want to set this)

Reply:
Done.